open_targets/data/discover/model.py

```diff
@@ -0,0 +1,103 @@
+"""Data models for Open Targets Platform data metadata in JSON."""
+
+from datetime import datetime
+from enum import Enum
+from typing import Annotated, Literal, TypeAlias
+
+from pydantic import BaseModel, ConfigDict, Field, field_validator
+from pydantic.alias_generators import to_camel
+
+
+class ConfiguredBaseModel(BaseModel):
+    """Base model with extra configuration.
+
+    Base model configured to convert names from camel case to snake case and
```

Comment:
It seems to me that the behaviour is the other way around, from snake case to camel case, no?

Reply:
Ah I think I got it now, the target *in Python* is snake case

Reply:
I improved the docstring a bit, I also renamed the directory from `discover` to `metadata`.